src/commonMain/kotlin/exchange.dydx.abacus/state/manager/V4StateManagerAdaptor+Transfer.kt

```diff
@@ -41,10 +45,11 @@ internal fun V4StateManagerAdaptor.retrieveDepositRoute(state: PerpetualState?)
         fromAmountString != null &&
         accountAddress != null &&
         chainId != null &&
+        dydxTokenDemon != null &&
```

Comment:
curious, what is the motivation for this check?

Reply:
The token denom comes from the env json. All fields are optional for json parsing